### PR TITLE
APP-80: C++ Storm demo compilation error fix

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @GamovCoder @dyosick

### CODE REVIEW

**Comment by ohamov** [comment]
LGTM

File: stormdemo/source/cpp/src/KaaDemo.cpp [hunk 1]
```diff
@@ -72,7 +72,7 @@ int main()
     auto kaaClient =  Kaa::newClient();
 
     // Set a custom strategy for uploading logs.
-    kaaClient->setLogUploadStrategy(std::make_shared<LogUploadStrategy>());
+    kaaClient->setLogUploadStrategy(std::make_shared<LogUploadStrategy>(kaaClient->getKaaClientContext()));
```
Comment (ghost) [inline stormdemo/source/cpp/src/KaaDemo.cpp]: Please, get rid of the LogUploadStrategy class and use RecordCountLogUploadStrategy(1, kaaClient->getKaaClientContext()) instead.
And add corresponding header.

**Comment by forGGe** [comment]
Looks good

**Comment by ghost** [comment]
LGTM

**Pull request merged**